helm/prometheus-rules/templates/alerting-rules/flux.rules.yml

```diff
@@ -148,7 +148,7 @@ spec:
           {{`Flux Image Automation Controller on {{ $labels.installation }} seems stuck.`}}
         opsrecipe: flux-image-automation-stuck/
       expr: |
-        sum(irate(workqueue_unfinished_work_seconds{name="imageupdateautomation",cluster_type="management_cluster",namespace="flux-system"}[15m])) > 0
+        sum(irate(workqueue_unfinished_work_seconds{name="imageupdateautomation",cluster_type="management_cluster",namespace=~"flux-giantswarm|flux-system"}[15m])) > 0
```

Comment:
Should you use namespace or exported namespace? Now I'm confused

Reply:
Flux exposed ones already have a namespace label. These are exposed by kubebuilder and they dont have it so the name is not mangled.

Reply:
I'm not sure I follow